api/v0/observe/observe.proto

```diff
@@ -52,7 +52,30 @@ service ObserveService {
   rpc GetPosixSignalsStream(GetPosixSignalsStreamRequest) returns (stream GetPosixSignalsStreamResponse) {}
 }
 
-message GetPosixSignalsStreamRequest { 
+/// Request a stream of POSIX signals
+message GetPosixSignalsStreamRequest {
+  /// The scope of this request. If no scope is specified, a stream of all POXIS
+  /// signals on the host will be returned.
+  oneof workload {
```

Comment:
this might generate nicer code as an enum and an opaque string "id".  oneof is pretty nasty, especially in some languages.

Reply:
We could introduce a "Workload" message with an enum and a string "id" and have this repeated. I need to think this thru a bit though what this means in terms of implementation details for this specific endpoint, as it could get pretty tricky (and not very intuitive) to aggregate POSIX signals for a mix of cells, containers, vms etc.

Reply:
What do you think about how the API looks now @dmah42? I will create a separate issue to discuss observing multiple workloads.

Reply:
Also, if we want to avoid using `oneof` b/c of how this gets compiled in different languages, should we capture this in the stdlib docs as a "general rule"? https://aurae.io/stdlib/

Reply:
we should probably point to https://google.aip.dev/ and specifically https://google.aip.dev/search?q=oneof which doesn't say avoid it, but it does provide some trade-offs.